jobs/build/oc_sync/Jenkinsfile

```diff
@@ -54,22 +55,25 @@ node {
 
 
     try {
-        currentBuild.displayName = "#${currentBuild.number} - ${FROM_RELEASE_TAG} - ${CLIENT_TYPE}"
+        currentBuild.displayName = "#${currentBuild.number} - ${RELEASE_NAME} - ${CLIENT_TYPE}"
+
+        def arch = commonlib.extractArchFromReleaseName(params.RELEASE_NAME)
```

Comment:
It looks like we are circling a lot by extracting arch from name, and building the tag from name + arch ... but conceptually now name and tag are different things ;D

Reply:
release name will have the arch in it if it's a nightly. it won't for named releases. when i looked at `oc_sync` it was more complicated than i was expecting to fix...